<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="newPatientButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>新建患者</string>
    </property>
   </widget>
   <widget class="QListWidget" name="patientListWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>81</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="patientInfoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="cameraPreviewLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopAllDataCollection">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>结束采集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startAllDataCollection">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>开始采集</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
